src/bentoml/_internal/models/skeleton_module.py

```diff
@@ -0,0 +1,18 @@
+import logging
```

Comment:
@MingLiangDai we can move `import logging` and `logger = ...` into `_log_warning_msg`. Because these functions are not supposed to be called, we do not need to import the logging module